### PR TITLE
package.json: Allow any version of node that supports the syntax

### PR DESCRIPTION
**Pull request by mjesuele**
I like this package a lot, and it saved me a bunch of time. Thanks!

There's no reason to exclude newer versions of Node in the engines field, and it causes errors for users of your package when installing. (I have to use `--ignoreEngines` with yarn when installing under Node 13, which is lame.) Anything from Node 8 on can run this code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,6 +83,6 @@
 		"react-dom": "^15.3.0 || ^16.0.0"
 	},
 	"engines": {
-		"node": "^12 || ^10 || ^8"
+		"node": ">= 8"
```
Comment (praneshr) [inline package.json]: Can you explicitly mention the node version to be supported? Something like below

```
"node": "8.x || 9.x || 10.x || 11.x || 12.x"
```

Comment (mjesuele) [reply package.json]: Why? The engines field is supposed to specify which versions of node can run your code. _Any_ version of newer than 8 will do. If you explicitly specify the major versions, you break yarn for users every time a new major version of node comes out, for no good reason.

**Comment by ciriarte** [comment]
@praneshr may I add another data point here, I also prefer the `>= 8` syntax unless there's a good reason not to, it forces my pipelines to keep track of the lowest common denominator across all my dependencies.

**Comment by mjesuele** [comment]
@praneshr Can you please merge this PR? This issue is incredibly annoying. We have to use `--ignoreEngines` every time we `yarn install` your package, like in our deployment commands or our CI pipeline. I think it should be telling that _no other package we use does something like this_.

**Pull request merged**